src/lib/zod/bridge_client_session.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod"
+
+export const bridge_client_session = z.object({
```

Comment:
We could actually start using the schemas from public types here, e.g., `import { schemas } from '@seamapi/types/connect` and then `schemas.bridge_client_session`. It may need to be added to the exported object first.

Don't hold this PR open on that change. More important to ensure we can use this on the bridge project.

Reply:
Yep, here it is https://github.com/seamapi/seam-connect/pull/10341